Description: Upstream changes introduced in version 0.1-0ubuntu1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 pluzzdl (0.1-0ubuntu1) oneiric; urgency=low
 .
   * Initial release
 .
 The person named in the Author field signed this changelog entry.
Author: Chaoswizard <chaoswizard@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- pluzzdl-0.1.orig/pluzzdl.sh
+++ pluzzdl-0.1/pluzzdl.sh
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-MAIN="__DATADIR__/pluzzdl/main.py"
+MAIN="/usr/share/pluzzdl/main.py"
 
 if [ ! -z "`python --version 2>&1 | grep 'Python 2'`" ]
 then
